ioztat

```diff
@@ -29,13 +29,15 @@
 
 import argparse
 import copy
+import dataclasses
 import math
 import os
 import re
 import shutil
 import signal
 import sys
 import time
+import typing
```

Comment:
Ditto. Though I'm not against adding type annotations if they have no runtime cost. Types are cool.